lms/views/admin/application_instance.py

```diff
@@ -309,6 +310,7 @@ def update_instance(self):
             ("vitalsource", "disable_licence_check", bool),
             ("jstor", "enabled", bool),
             ("jstor", "site_code", str),
+            ("hypothesis", "notes", str),
```

Comment:
Using a nested value inside `ai.settings`, this is both:

- The path of least resistance. Most of our settings setup expect this nested structure.
- Tidier to keep our own (as opposed to canvas/vitalsoruce/...) values in one place.